deployment-commons/pom.xml

```
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.mcollovati</groupId>
+        <artifactId>quarkus-hilla-parent</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>quarkus-hilla-commons-deployment</artifactId>
```

```suggestion
    <artifactId>quarkus-hilla-deployment-commons</artifactId>
```
I would switch the artifactId to have always the same naming scheme.

---

Having deployment suffix is the Quarkus default naming, but we can consider to change it.
However it will require more hard-coded strings in the pom instead of using properties

---

> Why does it require more hard-coded strings?

Because you cannot concatenate artifact id with the suffix with your suggested naming. You have to hard code the full deployment artifact name.
Anyway, not a big problem if we decide to go that way

---

Yea, I am actually not sure what is the best. Currently I am tending to go with the current naming.